packages/slice-machine/server/src/api/io/Slice.ts

```diff
@@ -10,3 +10,9 @@ export function readSlice(path: string): SliceSM {
 export function writeSlice(path: string, slice: SliceSM) {
   Files.write(path, Slices.fromSM(slice));
 }
+
+export function renameSlice(path: string, newSliceName: string) {
+  const slice = readSlice(path);
```

Comment:
The two functions above `renameSliece`, `readSlice` and `writeSlice` could be used (that way there will also be some runtime validation)

Reply:
I'm not sure what you mean here. I have used the `readSlice` and `writeSlice` in the renameSlice function 🤔

Reply:
neat, maybe I was looking in the wrong place :)